.travis.yml

```diff
@@ -1,21 +1,20 @@
-language: php
+version: ~> 1.0
 
-php: 5.6
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/cwp.yml
 
 env:
   global:
   - TRAVIS_NODE_VERSION="6"
```

Comment:
I don't think we need this variable. Let's use `.nvmrc` and `package.json` instead